src/aat/resources/features/F-097/S-424.td.json

```diff
@@ -0,0 +1,36 @@
+{
+	"_guid_": "S-424",
```

Comment:
You could define a base object for these test data objects and reuse common fields from it, like operation name, url, method, product name.

Reply:
Done.